src/Application/Commands/RetrospectivelyMatch.php

```diff
@@ -70,10 +86,25 @@ protected function doExecute(InputInterface $input, OutputInterface $output): in
 
         $numDistinctCampaigns = count($distinctCampaignIds);
 
```

Comment:
@bdsl I think until we have campaigns not in GBP this is OK. When we have others the logged output will be inaccurate, but I think it's only an issue in that reporting because the funds are agnostic to their underlying currency.